### PR TITLE
Add raw data link to footer

### PR DESCRIPTION
**Pull request by jonalport**
https://govausites.atlassian.net/browse/SC-87

Added in the the raw data footer link and tidied up the markup a little.

Copyright text colour updated to be a bit darker and still within contrast guidelines.

"Privacy policy" was splitting across two lines, so fixed that too.

New:

image

image

Old:

image

### CODE REVIEW

File: src/components/FooterContainer/Footer/Footer.js [hunk 1]
```diff
@@ -77,23 +77,31 @@ const Footer = props => (
...
+            {': '}
+            <a
+              href={`mailto:${EMAIL_ADDRESSES.CITY_PERFORMANCE}?subject=${STRINGS.CONTACT_SUBJECT}`}
+            >{EMAIL_ADDRESSES.CITY_PERFORMANCE}</a>
+          </div>
+
+          <div className={style.footerLinksRow}>
+            <a
+              href={LINKS.RAW_DATA}
+              target="_blank"
```
Comment (davidgilbertson) [inline src/components/FooterContainer/Footer/Footer.js]: Any link with `target="_blank"` must set `rel="noopener noreferrer"` to protect against [this vulnerability](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/) (however unlikely that is).

We've got a `<Link>` component that does this for you, but it also adds styling, so either adding `rel` or using `<Link>` is fine.

File: src/constants/misc.js [hunk 2]
```diff
@@ -67,4 +68,5 @@ export const STRINGS = {
   ALL_CITIES: 'All cities',
   CONTACT: 'Got a question on the Performance Framework? Contact PM&C',
   CONTACT_SUBJECT: 'Performance Framework',
+  RAW_DATA: 'View all raw data at dta.gov.au',
```
Comment (davidgilbertson) [inline src/constants/misc.js]: Should be `data.gov.au`, no?

**Review by davidgilbertson — CHANGES_REQUESTED** [review]
Two things

File: src/constants/misc.js (hunk 2 above)
Comment (jonalport) [reply src/constants/misc.js]: Yes that is probably right. I was following the designs! Also DTA...

File: src/components/Link/Link.js [hunk 3]
```diff
@@ -3,21 +3,35 @@ import PropTypes from 'prop-types';
...
+      rel="noopener noreferrer"
+      target="_blank"
+    >
+      {props.children || props.href}
+    </a>
+  );
+};
 
 Link.propTypes = {
+  unstyled: PropTypes.bool,
```
Comment (davidgilbertson) [inline src/components/Link/Link.js:27]: Nice.

File: src/components/Link/Link.js [hunk 4]
```diff
@@ -3,21 +3,35 @@ import PropTypes from 'prop-types';
...
   children: PropTypes.string,
   className: PropTypes.string,
   href: PropTypes.string.isRequired,
 };
 
+Link.defaultProps = {
+  unstyled: false,
```
Comment (davidgilbertson) [inline src/components/Link/Link.js:34]: Defaulting a prop to something falsey is redundant, but I'll allow it. (Unless you want to remove it and add a style section to the readme and put this in there as our very first rule.)

**Review by davidgilbertson — APPROVED** [review]
LGTM, with one optional thing.

**Pull request merged**